packages/cli/src/networks/index.ts

```diff
@@ -107,8 +106,8 @@ export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
   if (bootnodesFileUrl === null) {
     return [];
   }
-
-  const bootnodesFile = await got.get(bootnodesFileUrl).text();
+  const res = await fetch(bootnodesFileUrl);
+  const bootnodesFile = await res.text();
```

Comment:
this is not correct, if a 400/500 response is returned from the server then `res.text` contains the error response body / (or text).

We need to check if the request was successful via res.ok

Reply:
Sure. Thank!